spec/features/single_content_item_spec.rb

```diff
@@ -35,11 +35,6 @@
     it "renders the page without errors" do
       expect(page.status_code).to eq(200)
     end
-
-    it "does not display reading time" do
-      label = "About reading time"
-      expect(page).to_not have_selector("metric-summary__reading-time .govuk-details__summary-text", text: label)
-    end
```

Comment:
I'm not normally keen on deleting the entire test just because we're deleted a css class. However since it's testing for an absence of an element it's pretty prone to false positives anyway.